src/v/kafka/server/tests/BUILD

```diff
@@ -749,3 +749,101 @@ redpanda_cc_gtest(
         "@seastar//:testing",
     ],
 )
+
+redpanda_cc_bench(
+    name = "fetch_bench",
+    srcs = [
+        "fetch_bench.cc",
+    ],
+    args = [
+        "--duration=1",
+        "--runs=1",
+    ],
```

Comment:
Please remove this. I'll be adding a test target for benchmarks that does this automatically soon, but we don't want normal benchmarks to run like this :)